nncf/experimental/common/pruning/operations.py

```diff
@@ -171,19 +173,23 @@ def _both_dim_blocks_exist(left_idx, right_idx):
                 return True
             return False
 
-        output_mask = PropagationMask()
+        output_mask = None
+        dim_groups_map = {}
```

Comment:
Can we somehow interpretate empty `PropagationMask` as `None`?

Reply:
implemented `__bool__` for PropagationMask and handled in the code. 
Now PropagationMask with empty dictionary of groups considered as None, i.e. no pruning on this node